apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -2120,13 +2140,20 @@ static void libc_stdio_meminstream_tc(void)
 */
 static void libc_stdio_memoutstream_tc(void)
 {
-	FAR char buf[STDIO_BUFLEN];
+	char buffer[STDIO_BUFLEN];
+	char *buf = STREAM_TEST_CONTENTS;
 
 	struct lib_memoutstream_s memoutstream;
 
-	lib_memoutstream((FAR struct lib_memoutstream_s *)&memoutstream, buf, STDIO_BUFLEN);
-	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buffer, (FAR char *)(buf));
-	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buflen, (STDIO_BUFLEN - 1));	/* Save space for null terminator, hence checing with (STDIO_BUFLEN-1)*/
+	/* Check with valid length */
+
+	lib_memoutstream((FAR struct lib_memoutstream_s *)&memoutstream, buffer, STDIO_BUFLEN);
+	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buffer, (FAR char *)(buffer));
+	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buflen, (STDIO_BUFLEN - 1));	/* Save space for null terminator, hence checking with (STDIO_BUFLEN-1)*/
+
+	memoutstream.public.put((FAR struct lib_outstream_s *)&memoutstream.public, buf[0]);
+	TC_ASSERT_EQ("memoutstream_putc", memoutstream.public.nput, 1);
+	TC_ASSERT_EQ("lib_memoutstream", memoutstream.buffer[0], 'T');
```

Comment:
how about change 'T' to buf[0]?
If we change string later, we don't need to update all.